sdk/python/smithy/dbmanagers/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .remote_db_manager import RemoteDBManager
```

Comment:
why only exposing these classes to the outside? do we need to limit exposure to only these classes?